addons/website_sale/views/templates.xml

```diff
@@ -372,6 +373,13 @@
         </xpath>
     </template>
 
+    <!-- (Option) Products: Enable image full for all products : Images Full -->
+    <template id="products_images_full" name="Images Full" inherit_id="website_sale.products" active="False" customize_show="True">
+        <xpath expr="//td[@id='product_item']" position="attributes">
+            <attribute name="t-attf-class" add="#{request.website.viewref('website_sale.products_images_full').active and 'oe_image_full'}" separator=" "/>
```

Comment:
What is the value of `request.website.viewref('website_sale.products_images_full').active` when you are in this xpath?

Reply:
True indeed, thx